### ISSUE TITLE
Integrate with jvm-tools

### ISSUE THREAD

**Issue by nitsanw** [ajermakovics/jvm-mon#4]
Super cool! it's like htop for Java almost :-)
I would recommend having a look at this project:
https://github.com/aragozin/jvm-tools
In particular ttop would be a good drill down per process.

**Comment by ajermakovics** [comment]
Thanks!:) Great suggestion!
The bottom left part of the screen is not that useful at the moment so top thread's view would make more sense.
Looks like this is the entry point [ThreadTopCmd.java](https://github.com/aragozin/jvm-tools/blob/d4f482149ed718ed51faac88c4d661af435d24c0/sjk-core/src/main/java/org/gridkit/jvmtool/cmd/ThreadTopCmd.java#L44)

**Comment by nitsanw** [comment]
IMO the CPU gauge and heap bar are redundant, you got lots of room to play. Plotting the per thread/thread group/overall CPU usage and allocation rates is very useful

**Comment by ajermakovics** [comment]
Agreed. Version 0.2 has a view of top threads: https://github.com/ajermakovics/jvm-mon/releases/tag/0.2
At the moment I'm grabbing them from jvmtop library since that was already integrated. Eventually I'd like to integrate with jvm-tools since it seems to have more information like allocation rate.
